src/datepicker/docs/readme.md

```diff
@@ -1,145 +1,150 @@
+  The date object
+  
+* `shortcut-propagation`
+  _(Default: `false`)_ -
   An option to disable or enable shortcut's event propagation.
 
- * `template-url`
-  _(Default: 'uib/template/datepicker/datepicker.html')_ :
-  Allows overriding of default template of the datepicker
-
-
-### Popup Settings ###
-
-Options for datepicker can be passed as JSON using the `datepicker-options` attribute.
-Specific settings for the `uib-datepicker-popup`, that can globally configured through the `uibDatepickerPopupConfig`, are:
-
- * `uib-datepicker-popup`
- 	_(Default: 'yyyy-MM-dd')_ :
- 	The format for displayed dates. This string can take string literals by surrounding the value with single quotes, i.e. `yyyy-MM-dd h 'o\'clock'`
-
- * `show-button-bar`
- 	_(Default: true)_ :
- 	Whether to display a button bar underneath the datepicker.
-
- * `current-text`
- 	_(Default: 'Today')_ :
- 	The text to display for the current day button.
-
- * `clear-text`
- 	_(Default: 'Clear')_ :
- 	The text to display for the clear button.
-
- * `close-text`
- 	_(Default: 'Done')_ :
- 	The text to display for the close button.
-
- * `close-on-date-selection`
- 	_(Default: true)_ :
- 	Whether to close calendar when a date is chosen.
-
- * `datepicker-popup-template-url`
-  _(Default: 'uib/template/datepicker/popup.html')_ :
-  Allows overriding of default template of the popup
-
- * `datepicker-template-url`
-  _(Default: 'uib/template/datepicker/datepicker.html')_ :
-  Allows overriding of default template of the datepicker used in popup
-
- * `datepicker-append-to-body`
-  _(Default: false)_:
-  Append the datepicker popup element to `body`, rather than inserting after `datepicker-popup`. For global configuration, use `uibDatepickerPopupConfig.appendToBody`.
-
- * `is-open` <i class="glyphicon glyphicon-eye-open"></i>
-  _(Default: false)_:
+* `show-weeks`
+  _(Default: `true`)_ -
+  Whether to display week numbers.
+
+* `starting-day`
+  _(Default: `0`)_ -
+  Starting day of the week from 0-6 (0=Sunday, ..., 6=Saturday).
+  
+* `template-url`
+  _(Default: `uib/template/datepicker/datepicker.html`)_ -
+  Add the ability to override the template used on the component.
+
+* `year-range`
+  _(Default: `20`)_ -
+  Number of years displayed in year selection.
+
+### uib-datepicker-popup settings ###
+
+Options for the uib-datepicker must be passed as JSON using the `datepicker-options` attribute. This list is only for popup settings.
+
+* `clear-text`
+  _(Default: `Clear`)_ -
+  The text to display for the clear button.
+  
+* `close-on-date-selection`
+  _(Default: `true`)_ -
+  Whether to close calendar when a date is chosen.
+  
+* `close-text`
+  _(Default: `Done`)_ -
+  The text to display for the close button.
+  
+* `current-text`
+  _(Default: `Today')_ -
+  The text to display for the current day button.
+  
+* `datepicker-append-to-body`
+  _(Default: `false`)_ -
+  Append the datepicker popup element to `body`, rather than inserting after `datepicker-popup`.
```

Comment:
Do we want to add a blurb as to when it would be a good idea for the user to use this?